suite_cam6.xml

```diff
@@ -0,0 +1,361 @@
+        <scheme>zt2zm_convert</scheme>                <!-- Transition variables on thermodynamic levels to momentum levels (manageable by registry/meta file?) -->
+        <scheme>clubb_var_vert_inv</scheme>           <!-- Change order of vertical levels for CLUBB input variables (should be manageable via the registry/meta file) -->
+        <subcycle loop="clubb_time_substep">          <!-- CLUBB sub-step loop (number of iterations should be calculated in the "clubb_dtime_calc" scheme), CAM6 var = 'clubb_nadv' -->
+          <scheme>stats_begin_timestep_api</scheme>   <!-- Only matters if "l_stats" is True -->
+          <scheme>advance_clubb_core_api</scheme>     <!-- Actual CLUBB parameterization scheme -->
+          <scheme>update_xp2_mc_api</scheme>          <!-- Only matters if "do_rainturb" is True -->
+          <scheme>calculate_thlp2_rad_api</scheme>    <!-- Only matters if "do_cldcool" is True -->
+          <scheme>stats_end_timestep_clubb</scheme>   <!-- Only matters if "l_stats" is True -->
+        </subcycle>
+        <scheme>zm2zt_convert</scheme>                <!-- Transition variables on momentum levels to thermodynamic levels (manageable by registry/meta file?) -->
+        <scheme>clubb_var_vert_rev</scheme>           <!-- Change order of vertical levels for CLUBB output variables back to host model ordering  (should be manageable via the registry/meta file) -->
+        <scheme>clubb_upper_diss</scheme>             <!-- CLUBB energy and total water mass fixer -->
+        <scheme>clubb_cam_update</scheme>             <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+        <scheme>liquid_macro_tend</scheme>            <!-- Only matters if "clubb_do_liqsupersat" is True-->
+        <scheme>liquid_macro_CAM_update</scheme>      <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+        <scheme>conv_cond_detrain_calc</scheme>       <!-- Calculate the host model cloud condensate and energy tendencies produced by convective detrainment -->
+        <scheme>conv_cond_detrain_cam_update</scheme> <!-- Placeholder for CAM ptend and diagnostic output calculations -->
+        <scheme>set_wet_to_dry</scheme>               <!-- Convert mixing ratios from air + water vapor to dry air -->
+        <scheme>clubb_diag_output</scheme>            <!-- Calculate numerous different CLUBB diagnostic variables, and output them to history file -->
+        <!-- **************************** -->
+        <scheme>check_energy_chng</scheme>            <!-- Global integral checker required for certain diagnostic outputs -->
+        <subcol gen="subcol_generator_name" avg="subcol_averager_name">
+          <process_split>
+            <!-- MG aerosol microphysics -->
+            <!-- **************************** -->
+            <scheme>microp_aero_run</scheme>          <!-- Some, but not all, tendencices from here are added to the state before microp.  This is done via pbuf.-->
```

Comment:
The short answer is because I considered `mircop_aero_run` to be a parameterizaton, as opposed to an interface to a parameterization (whereas `clubb_tend_cam` is clearly an interface to the CLUBB paramterization).  

However, I do agree that these sort of decisions are fairly subjective, and so I am happy to discuss where the granularity "cut-off" should be for any particular physics parameterization/interface.

Reply:
I believe that it needs to be broken up as it makes it really hard to switch other parts of the code (e.g, some desired uses of CARMA) to match. I understand that this is tricky right now because, for instance, what should you do with all the `rad_cnst_xxx` stuff? We could postpone this for now and add a comment.

Reply:
I went ahead and split the `microp_aero_run` scheme into its corresponding sub-sections and subroutines.  I imagined one could combine all of the `rad_cnst_xxx` calls into a single `aero_get_num_mmr` scheme, so I've gone ahead and done so, at least for now.